website/src/ts/scripts/slds-components.ts

```diff
@@ -1,5 +1,6 @@
 import '@oddbird/popover-polyfill';
 import '@webcomponents/scoped-custom-element-registry/scoped-custom-element-registry.min.js';
+import '@sl-design-system/shared';
```

Comment:
Does this import make sense? Why do we need it?

Reply:
No, I forgot to remove it :)